### PR TITLE
@oxaudo => Redirect to auction registration when `Register` clicked

### PR DESCRIPTION
**Pull request by mzikherman**
This seems pretty easy!

I'll add the bid one next + tests.

### CODE REVIEW

**Comment by mzikherman** [comment]
Ok, I think this might be mergeable! We can tackle adding tests to this component overall in a separate PR.

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx [hunk 1]
```diff
@@ -12,19 +12,32 @@ export interface ArtworkSidebarBidActionProps {
...
+
+  redirectToRegister = () => {
+    const { sale } = this.props.artwork
+    window.location.href = `${sd.APP_URL}/auction-registration/${sale.id}`
+  }
+
+  redirectToBid = (firstIncrement: number) => {
+    const { id, sale } = this.props.artwork
+    const bid = this.state.selectedMaxBidCents || firstIncrement
+    window.location.href = `${sd.APP_URL}/auction/${
```
Comment (oxaudo) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx]: I hope we do not need to worry about user presence here and the bid url will take care of that.

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx [hunk 2]
```diff
@@ -59,7 +72,12 @@ export class ArtworkSidebarBidAction extends React.Component<
       return (
         <Box>
           {!registrationAttempted && (
-            <Button width="100%" size="medium" mt={1}>
+            <Button
+              width="100%"
+              size="medium"
+              mt={1}
+              onClick={() => this.redirectToRegister()}
```
Comment (oxaudo) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx]: Also wondering if we should register any analytics event on those buttons.

**Comment by oxaudo** [comment]
This looks good except for the conflict in generated file. Which is I'm not even sure why it's updated...

Also have couple of unrelated thoughts but merge this once updated plz.

**Pull request merged**